cmd/spire-server/cli/run/run.go

```diff
@@ -315,6 +316,14 @@ func newServerConfig(c *config) (*server.Config, error) {
 	sc.ProfilingFreq = c.Server.ProfilingFreq
 	sc.ProfilingNames = c.Server.ProfilingNames
 
+	if c.Server.MaxConnectionAge != "" {
```

Comment:
I would add a minimum here so that someone doesn't mistakenly think this is in "seconds" and set their maxconnectionage to 60 nanoseconds.

Reply:
I'm not sure that sort of mistake would happen since the unit suffix is required (i.e. `60ns` v.s. `60s`).

Reply:
I had not read the ParseDuration docs, I agree.